doc/source/development/contributing_environment.rst

```diff
@@ -149,9 +149,10 @@ Step 4: build and install Xorbits
 You can now run::
 
    # Build and install Xorbits
+   python -m pip install -e ".[dev]"
    python setup.py build_ext -i
+   # install nodejs  >= 14
```

Comment:
This comment is not necessary.

Xorbits requires node 18 or higher, please see: https://doc.xorbits.io/en/latest/development/contributing_environment.html#id2

Reply:
Got it.